app/scripts/views/sign_up.js

```diff
@@ -298,6 +309,17 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
             return;
           }
 
+          // if we are setting up Firefox Sync and what the user to choose what to sync first
+          if (self.relier.isSync() && self._isSyncChooser()) {
```

Comment:
Ahha, the checkbox is only displayed if the browser does not have that capability.